src/20250519_Multi_GHOAvalancheLaunch/AaveV3Avalanche_GHOAvalancheListing_20250519.t.sol

```diff
@@ -0,0 +1,334 @@
+ */
+contract AaveV3Avalanche_GHOAvalancheListing_20250519_Base is ProtocolV3TestBase {
+  AaveV3Avalanche_GHOAvalancheListing_20250519 internal proposal;
+
+  ITokenAdminRegistry internal constant TOKEN_ADMIN_REGISTRY =
+    ITokenAdminRegistry(GHOAvalancheLaunch.AVAX_TOKEN_ADMIN_REGISTRY);
+  address internal constant ROUTER = GHOAvalancheLaunch.AVAX_CCIP_ROUTER;
+  address internal constant RMN_PROXY = GHOAvalancheLaunch.AVAX_RMN_PROXY;
+  address internal constant RISK_COUNCIL = GHOAvalancheLaunch.RISK_COUNCIL;
+  IGhoToken internal constant GHO_TOKEN = IGhoToken(GHOAvalancheLaunch.GHO_TOKEN);
+  address internal constant NEW_REMOTE_POOL_ARB = GhoArbitrum.GHO_CCIP_TOKEN_POOL;
+  address internal constant NEW_REMOTE_POOL_ETH = GhoEthereum.GHO_CCIP_TOKEN_POOL;
+  address internal constant NEW_REMOTE_POOL_BASE = GhoBase.GHO_CCIP_TOKEN_POOL;
+  IGhoAaveSteward internal constant NEW_GHO_AAVE_STEWARD =
+    IGhoAaveSteward(GHOAvalancheLaunch.GHO_AAVE_CORE_STEWARD);
+  IGhoBucketSteward internal constant NEW_GHO_BUCKET_STEWARD =
+    IGhoBucketSteward(GHOAvalancheLaunch.GHO_BUCKET_STEWARD);
+  IGhoCcipSteward internal constant NEW_GHO_CCIP_STEWARD =
+    IGhoCcipSteward(GHOAvalancheLaunch.GHO_CCIP_STEWARD);
+  IUpgradeableBurnMintTokenPool_1_5_1 internal constant NEW_TOKEN_POOL =
+    IUpgradeableBurnMintTokenPool_1_5_1(GHOAvalancheLaunch.GHO_CCIP_TOKEN_POOL);
+
+  function setUp() public virtual {
+    vm.createSelectFork(vm.rpcUrl('avalanche'), GHOAvalancheLaunch.AVAX_BLOCK_NUMBER);
+    proposal = new AaveV3Avalanche_GHOAvalancheListing_20250519();
+  }
+
+  function _executeLaunchAIP() internal {
+    executePayload(vm, address(new AaveV3Avalanche_GHOAvalancheLaunch_20250519()));
+  }
+
+  function _mockBridgeSeedAmount() internal {
+    uint256 seedAmount = proposal.GHO_SEED_AMOUNT();
+    vm.prank(address(NEW_TOKEN_POOL));
+    GHO_TOKEN.mint(GovernanceV3Avalanche.EXECUTOR_LVL_1, seedAmount);
+  }
+
+  function _isDifferenceLowerThanMax(
+    uint256 from,
+    uint256 to,
+    uint256 max
+  ) internal pure returns (bool) {
+    return from < to ? to - from <= max : from - to <= max;
+  }
+
+  function assertEq(
+    IRateLimiter.TokenBucket memory bucket,
+    IRateLimiter.Config memory config
+  ) internal pure {
+    assertEq(bucket.capacity, config.capacity);
+    assertEq(bucket.rate, config.rate);
+    assertEq(bucket.isEnabled, config.isEnabled);
+  }
+
+  function assertEq(
+    IDefaultInterestRateStrategyV2.InterestRateData memory a,
+    IDefaultInterestRateStrategyV2.InterestRateData memory b
+  ) internal pure {
+    assertEq(a.optimalUsageRatio, b.optimalUsageRatio);
+    assertEq(a.baseVariableBorrowRate, b.baseVariableBorrowRate);
+    assertEq(a.variableRateSlope1, b.variableRateSlope1);
+    assertEq(a.variableRateSlope2, b.variableRateSlope2);
+  }
+}
+
+contract AaveV3Avalanche_GHOAvalancheListing_20250519_ListingPreRequisites is
```

Comment:
missing default test, so that it generates the diffs md, which will indicate what will change when executing the payload. should be added on the other networks also